common/djangoapps/student/views.py

```diff
@@ -1541,7 +1541,7 @@ def user_signup_handler(sender, **kwargs):  # pylint: disable=unused-argument
             log.info(u'user {} originated from a white labeled "Microsite"'.format(kwargs['instance'].id))
 
 
-def _do_create_account(form, custom_form=None):
+def _do_create_account(form, custom_form=None, site=None):
```

Comment:
@douglashall Does this mean you are ok with the attribute here to have `site_id=None` written into our production database, and understand that is coming from the management command? Sound very implicit and documentation won't help explain in the future.